app/models/BusinessSearchRequest.scala

```diff
@@ -0,0 +1,16 @@
+package models
+
+import play.api.mvc.{ AnyContent, Request }
+
+import scala.util.Try
+
+case class BusinessSearchRequest(term: String, offset: Int, limit: Int, suggest: Boolean, defaultOperator: String)
+
+object BusinessSearchRequest {
+  def apply(term: String, request: Request[AnyContent], suggest: Boolean = false) = {
+    val offset = Try(request.getQueryString("offset").get.toInt).getOrElse(0)
```

Comment:
We should only need the Try to protect the type conversion - using get is a bit sloppy (and is getting flagged by the style checker).

option.flatMap(str => Try(str.toInt).toOption).getOrElse(...)